addons/l10n_latam_check/wizards/l10n_latam_payment_mass_transfer.py

```diff
@@ -35,29 +36,36 @@ def default_get(self, fields_list):
         if self._context.get('active_model') != 'account.payment':
             raise UserError(_("The register payment wizard should only be called on account.payment records."))
         payments = self.env['account.payment'].browse(self._context.get('active_ids', []))
```

Comment:
we added the if condition. However, the check_ids field is not in the fields_list. As far as I see is because the field was not in view. I added the check_ids field as invisible to the view and now the check_ids is part of the fields_list